papas/aliceproto/aliceblockbuilder.py

```diff
@@ -0,0 +1,215 @@
+
+import itertools
+from PFfloodfill import FloodFill
+from enum import Enum
+#import sys
+#Get the DAGtool
+
+from DAG import Node, BreadthFirstSearchIterative,DAGFloodfill
+from heppy.papas.aliceproto.Identifier import Identifier
+from heppy.papas.pfalgo.distance import Distance
+ 
+class Event(object):
```

Comment:
why the need for an event class? I understood that we would use the event that is passed to all heppy modules